.github/workflows/acceptance-tests.yml

```diff
@@ -85,6 +85,8 @@ jobs:
       azure_key_vault_name_updated: ${{ secrets.AZURE_KEY_VAULT_NAME_UPDATED }}
       azure_key_identifier_updated: ${{ secrets.AZURE_KEY_IDENTIFIER_UPDATED }}
       azure_app_secret: ${{ secrets.AZURE_APP_SECRET }}
+      confluent_cloud_api_key: ${{ secrets.CONFLUENT_CLOUD_API_KEY }}
```

Comment:
Any concern on using the same CONFLUENT_CLOUD in both `dev`/`qa`?

Reply:
We have only access to this Confluent, which is a prod environment as far as I know, I don't think there's any concern on using the same for QA/DEV. One thing I am trying is to avoid creating a confluent cluster in the tests to reduce the usage of confluent. Will update the PR if I manage to run the tests without the cluster